### ISSUE TITLE
Result of encoder is not used

### ISSUE THREAD

**Issue by lll000111** [wkh237/react-native-fetch-blob#551]
https://github.com/wkh237/react-native-fetch-blob/blob/master/android/src/main/java/com/RNFetchBlob/RNFetchBlobFS.java#L269

The encoder returns a buffer - which is ignored.

https://docs.oracle.com/javase/7/docs/api/java/nio/charset/CharsetEncoder.html#encode(java.nio.CharBuffer)

@wkh237 I think the returned buffer should be used in the following steps instead of the original buffer?

**Comment by NamanArora** [comment]
Can we work upon issues labeled 'bug' and provide a pr?

**Comment by lll000111** [comment]
Of course. Not sure when they get merged, the RNFB owner seems to be under water. You can always maintain a separate branch.
